rendering_resource_manager_service/service/settings.py

```diff
@@ -176,6 +176,7 @@
 # Slurm credentials (To be modified by deployment process)
 SLURM_USERNAME = 'TO_BE_MODIFIED'
 SLURM_KEY = 'TO_BE_MODIFIED'
+SLURM_PROJECT = 'TO_BE_MODIFIED'
```

Comment:
Good point, I was actually thinking of setting them to an empty value so that they would automatically be ignored, but I think it is better to generate an error in case users forget to set those values in the first place.